modules/rostests/kmtests/kmtest/support.c

```diff
@@ -141,15 +142,21 @@ KmtLoadDriver(
 
     if (Error)
     {
+        ok(FALSE, "KmtCreateAndStartService(\"%ls\") failed! (Error = 0x%08lx)\n", TestServiceName, Error);
+
         // TODO
         __debugbreak();
```

Comment:
The TODO means "properly handle failure here".
This may be as simple as printing an error message and returning.
Or if callers rely on this function always succeeding, it may require either a call to `exit` or refactoring this to return an error code and adding error handling in the callers.